src/lib.rs

```diff
@@ -184,13 +189,15 @@ impl<'a> Nlmsg<'a> {
     }
 }
 
+#[derive(Debug, Eq, PartialEq)]
```

Comment:
I wouldn't derive `PartialEq` for the `PayloadState`, because we never need to compare Vec<u8> of the Owned state, and logically such comparison doesn't make much sense.

Reply:
This relates to the change on line 843, but I can revert both if you prefer:
```rust
if msg.verdict != Verdict::Drop && msg.payload_state != PayloadState::Unmodified`
 ```

Reply:
I'd like both to be reverted. Thanks.